superset/config.py

```diff
@@ -790,7 +790,7 @@ class CeleryConfig:  # pylint: disable=too-few-public-methods
 # chrome:
 #   Requires: headless chrome
 #   Limitations: unable to generate screenshots of elements
-EMAIL_REPORTS_WEBDRIVER = "firefox"
+WEBDRIVER_TYPE = "firefox"
```

Comment:
optional: change default to `"chrome"` ?

Reply:
I would, but: https://github.com/apache/incubator-superset/blob/master/docs/installation.rst#email-reports

States that `firefox` is preferred, is it reasonable to change the default?

Reply:
I believe there are unresolved rendering errors on Firefox. Might not be applicable for thumbs, but worth mentioning.